developers/contracts/btc-peg-out-endpoint-v2-01.md

```diff
@@ -0,0 +1,151 @@
+# btc-peg-out-endpoint-v2-01.clar
+
+- Location: `xlink/packages/contracts/bridge-stacks/contracts/btc-peg-out-endpoint-v2-01.clar`
+- [Deployed contract]()
+
+This technical document provides a detailed overview of the contract responsible for managing the peg-out process, enabling the transfer of bridged BTC from the Stacks network back to the Bitcoin network. In this process, aBTC (Bridged BTC tokens on Stacks) is burned or transferred, and BTC is released to a specified Bitcoin address. The contract's primary functionality is implemented through a series of public functions. Let's review this core operation.
```

Comment:
About the comment above it seems that the contracts does both things according to different situactions. Because of that, perhaps we can clarify above that sometimes it does one thing and sometimes the other... (burn vs transfer..)  But feel free to chose the one you consider the best (adding that info or not)